pyproject.toml

```diff
@@ -23,7 +23,7 @@ classifiers = [
     "Development Status :: 2 - Pre-Alpha",
     "Framework :: napari",
     "Intended Audience :: Developers",
-    "License :: OSI Approved :: BSD License",
```

Comment:
Trove classifiers must come from [the shared list](https://pypi.org/classifiers/). For the AISL, the following seem appropriate:
```
License :: Other/Proprietary License
License :: Free for non-commercial use
```

Reply:
ca56c801123e80e8574dbf6d4e7b942b4bb91d91